src/cargo/util/toml/mod.rs

```diff
@@ -2393,8 +2408,8 @@ fn inheritable_from_path(
     }
 }
 
-/// Returns the name of the README file for a `TomlProject`.
-pub fn readme_for_project(package_root: &Path, readme: Option<StringOrBool>) -> Option<String> {
+/// Returns the name of the README file for a `TomlPackage`.
```

Comment:
Would it be better with intra doc link

```suggestion
/// Returns the name of the README file for a [`TomlPackage`].
```